### ISSUE TITLE
 Inquiry about Multiple-Point Geostatistical Methods for Continuous Variable Modeling

### ISSUE THREAD

**Issue by Alvin-prc** [randlab/geone#11]
I am  a newcomer to the field of multiple-point geostatistics with a keen interest in understanding the various methods used in this domain. I have come across terms such as SNESIM, GENESIM, and DeeSse, which seem to represent different approaches within multiple-point geostatistics.

My understanding is that multiple-point geostatistical algorithms are commonly used to characterize reservoir structures. As reservoir characterization often involves dealing with categorical variables, I am curious to know if these methods can be extended to model continuous variables, specifically in establishing a continuous permeability field for a given site.

I would greatly appreciate it if you could provide insights on the following questions:

What are the other types of multiple-point geostatistical methods, and to which category does direct sampling belong? Which method is considered more practical?

Can multiple-point geostatistical methods be utilized to construct a continuous permeability field?

If so, are the above-mentioned methods (SNESIM, GENESIM, DeeSse) applicable to modeling continuous variables, or does their suitability depend on the algorithm used or the training image employed?

**Comment by algocompretto** [comment]
>  I am curious to know if these methods can be extended to model continuous variables

Yes, some methods are specifically designed for categorical variables, but DeeSse and Direct Sampling can be applied to continuous variables as well.

> What are the other types of multiple-point geostatistical methods, and to which category does direct sampling belong? Which method is considered more practical?

There are no silver bullet that solves all the problems! It is important to evaluate each algorithm individually for different cases to determine which one is more practical. You can find more information about MPS methods in this [link](https://link.springer.com/chapter/10.1007/978-3-319-78999-6_30)



> Can multiple-point geostatistical methods be utilized to construct a continuous permeability field?

Yes, multiple-point geostatistical methods can be used to construct a continuous permeability field.

I hope this explanation has provided you with the information you were seeking!
